build/VSCompatibleLayer.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <PropertyGroup>
+    <Platforms>AresDX;AresGL;AresXNA;TSDX;TSGL;TSXNA;YRDX;YRGL;YRXNA</Platforms>
```

Comment:
See comment on solution file. We should keep the existing platform values.

Also from [Common MSBuild project properties](https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2022):
`Platform:
The operating system you are building for. Examples for .NET Framework builds are "Any CPU", "x86", and "x64".'`